src/main/index.ts

```diff
@@ -691,7 +678,19 @@ const router = t.router({
   }),
   handleSetupAndLaunch: handlePasswordInput(handleSetupAndLaunch),
   launch: handlePasswordInput(handleLaunch),
-  getAppPort: t.procedure.query(() => APP_PORT),
+  getAppPortAndIsDevHubInstalled: t.procedure.query(() => ({
```

Comment:
What's the motivation to conflate the app port and the `isDevhubInstalled` into a single query? I'm not categorically against it but it seems to me like that's information that does not necessarily belong together.

Reply:
Especially since there is already a separate query `isDevhubInstalled`

Reply:
fixed